app/src/main/java/com/github/sdp/tarjetakuna/MainActivity.kt

```diff
@@ -33,7 +33,7 @@ class MainActivity : AppCompatActivity() {
         }
         val drawerLayout: DrawerLayout = binding.drawerLayout
         val navView: NavigationView = binding.navView
-        val navController = findNavController(R.id.nav_host_fragment_content_drawer)
+        val navController = (supportFragmentManager.findFragmentById(R.id.nav_host_fragment_content_drawer) as NavHostFragment).navController
```

Comment:
For other reviewers: this is because findNavController requires the drawer to already be created (which it isn't in onCreate), when FragmentContainerView does not.
https://issuetracker.google.com/issues/142847973?pli=1

Reply:
Good catch btw Kelvin. :)